openlibrary/core/ia.py

```diff
@@ -1,6 +1,5 @@
 """Library for interacting wih archive.org.
 """
-from __future__ import print_function
```

Comment:
why was this removed? Because the print functions were removed? (this was for python 3 compat.)

Reply:
There are no uses of print in the module.